website/docs/r/file_instance.html.markdown

```diff
@@ -0,0 +1,141 @@
+---
+# ----------------------------------------------------------------------------
+#
+#     ***     AUTO GENERATED CODE    ***    AUTO GENERATED CODE     ***
+#
+# ----------------------------------------------------------------------------
+#
+#     This file is automatically generated by Magic Modules and manual
+#     changes will be clobbered when the file is regenerated.
+#
+#     Please read more about how to change this file in
+#     .github/CONTRIBUTING.md.
+#
+# ----------------------------------------------------------------------------
+layout: "google"
+page_title: "Google: google_file_instance"
+sidebar_current: "docs-google-file-instance"
+description: |-
+  A Google Cloud Filestore instance.
+---
+
+# google\_file\_instance
+
+A Google Cloud Filestore instance.
+
+To get more information about Instance, see:
+
+* [API documentation](https://cloud.google.com/memorystore/docs/redis/reference/rest/)
+* How-to Guides
+    * [Official Documentation](https://cloud.google.com/memorystore/docs/redis/)
+
+## Example Usage
+
+### Basic Usage
+```hcl
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+
+* `name` -
+  (Required)
+  The resource name of the instance, in the format
+  `projects/{project_id}/locations/{locationId}/instances/{instanceId}`.
```

Comment:
Is this actually the case? I think we allow just {instanceId}, and if not, we should